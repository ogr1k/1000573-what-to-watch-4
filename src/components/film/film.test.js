import React from "react";
import renderer from "react-test-renderer";
import Film from "./film.jsx";

const film = {
  name: `Jojo Rabbit`,
  poster: `https://images-na.ssl-images-amazon.com/images/I/71DIYFHebjL._AC_SY741_.jpg`,
  description: ``,
  director: ``,
  genre: ``,
  rating: 1,
  ratings: 1,
  starring: ``,
  year: 1
};

it(`Render Film`, () => {
  const tree = renderer
    .create(<Film film={film} onClick={() => {}} onmouseover={() => {}}
      onmouseout={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
