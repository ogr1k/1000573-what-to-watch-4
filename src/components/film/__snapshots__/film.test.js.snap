// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Film 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <img
      alt="Jojo Rabbit"
      height="175"
      src="https://images-na.ssl-images-amazon.com/images/I/71DIYFHebjL._AC_SY741_.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
    >
      Jojo Rabbit
    </a>
  </h3>
</article>
`;
